speed-dreams/source-2.2.3/data/data/menu/soundconfigmenu.xml

```diff
@@ -257,6 +268,17 @@
 		<attstr name="font" val="small_t"/>
 	</section>
 
+    <section name="8">
+        <attstr name="type" val="label"/>
+        <attnum name="x" val="20"/>
+        <attnum name="y" val="200"/>
```

Comment:
```suggestion
        <attnum name="y" val="203"/>
```
y-pos is misaligned (all labels are 3 higher than the one they are for, that way they are centered)